elip-0100.mediawiki

```diff
@@ -64,15 +64,17 @@ This specification relies on definition of roles in the main PSET standard. The
 
 ===Updater===
 
-Besides the responsibilities defined in PSET specification, Updater should add asset metadata for each asset referenced in any of inputs and outputs. This information is typically queried from the global asset registry, but cached or pre-defined information could be used as well. For each asset, the Updater adds a separate <tt>PSBT_ELEMENTS_HWW_GLOBAL_ASSET_METADATA</tt> field using asset tag as keydata. It is desirable however not critical for the Updater to verify asset metadata if it is received from an untrusted source. This verification is obligatory for the Signer role.
+Besides the responsibilities defined in PSET specification, Updater should add asset metadata for each asset referenced in any of inputs and outputs. This information is typically queried from the global asset registry, but cached or pre-defined information could be used as well. For each asset, the Updater adds a separate field using asset tag as keydata. Updater is responsible to make a reasonable number of checks for the asset metadata before it is injected in PSET intended for offline signing. These checks must include verification of metadata source. It is also critical for Updater to perform the best effort test to detect the bogus assets impersonating legitimate assets through visually or semantically similar names and tickers.
 
 As a single entity is likely to be both a Creator and Updater, and thus it makes possible to fill-in asset metadata on PSET creation step.
 
+<tt>WARNING:</tt> PSET containing asset metadata is not supposed to be transmitted to other parties to not encourage them to trust this information. Updater must remove all asset metadata records after completing the signature process involving an offline entity.
```

Comment:
nit: `other parties to not encourage them to trust this information` -> `other parties, to avoid encouraging them to trust this information`.